Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -741,10 +721,11 @@ void Wiimote::LoadDefaults(const ControllerInterface& ciface)
   // B
   m_buttons->SetControlExpression(1, "`Click 1`");
 #endif
-  m_buttons->SetControlExpression(2, "`1`");     // 1
-  m_buttons->SetControlExpression(3, "`2`");     // 2
-  m_buttons->SetControlExpression(4, "Q");       // -
-  m_buttons->SetControlExpression(5, "E");       // +
+  // 1 2 - +
+  m_buttons->SetControlExpression(2, "`1`");
+  m_buttons->SetControlExpression(3, "`2`");
+  m_buttons->SetControlExpression(4, "Q");
+  m_buttons->SetControlExpression(5, "E");
```

Comment:
This change seems unrelated?

Reply:
There's a clang-format version mismatch or something and the lint-bot wants a different number of spaces between the `;` and the `//` than what my clang-format wants. I keep touching this file and having to undo the re-formatting to that section to make the lint-bot happy so I just moved the comment up there.